vaadin-dialog-flow/src/main/java/com/vaadin/flow/component/dialog/Dialog.java

```diff
@@ -68,7 +68,8 @@ public Dialog() {
         container.getClassList().add("draggable-leaf-only");
         container.getStyle().set(ElementConstants.STYLE_WIDTH, "100%");
         container.getStyle().set(ElementConstants.STYLE_HEIGHT, "100%");
-        container.getStyle().set("display", "inline-block");
+        container.getStyle().set("display", "flex");
+        container.getStyle().set("flex-direction", "column");
```

Comment:
This causes a (breaking) change for the layout: The "Confirmation dialog" demo buttons align differently (on top of each other) after the change:
![Screenshot 2020-09-30 at 9 38 50](https://user-images.githubusercontent.com/1222264/94652716-2a09c280-0303-11eb-94e0-6588cd84949b.png)

Reply:
...it depends on the type of the items. Block elements should layout on top of each other so flexbox might be problematic.

Reply:
Yeah. I was thinking about that late yesterday. Using `flex`, we will either force block elements to align next to each other (`flex-direction: row`) or inline elements to align on top of each other (`flex-direction: column`).

Reply:
So are there some alternatives to using flex for the container? Or would it fix the issue to make the parent flow-component-renderer element a flexbox?

Reply:
I started trying to apply changes to `flow-component-renderer`. Now trying to set `flex` there. It looks as it is working locally. Please, take a look.